wire-ios-sync-engine/Source/Synchronization/IncrementalSyncObserver.swift

```diff
@@ -55,7 +55,8 @@ final class IncrementalSyncObserver: IncrementalSyncObserverProtocol {
                 .receive(on: decryptionQueue)
                 .sink { [weak self] syncState in
                     switch syncState {
-                    case .incrementalSyncing(.pullPendingEvents):
+                    case .incrementalSyncing(.pullPendingEvents),
+                         .incrementalSyncing(.receivingLiveEvents):
```

Comment:
note: this is to differentiate IncrementalSyncV2 doing quickSync so we should not be allowed to send messages.